test/srt/test_nightly_human_eval.py

```diff
@@ -0,0 +1,125 @@
+import os
+import shutil
+import signal
+import subprocess
+import unittest
+from types import SimpleNamespace
+
+from sglang.srt.utils import kill_child_process
+from sglang.test.test_utils import (
+    DEFAULT_MODEL_NAME_FOR_NIGHTLY_EVAL_FP8_TP1,
+    DEFAULT_MODEL_NAME_FOR_NIGHTLY_EVAL_FP8_TP2,
+    DEFAULT_MODEL_NAME_FOR_NIGHTLY_EVAL_TP1,
+    DEFAULT_MODEL_NAME_FOR_NIGHTLY_EVAL_TP2,
+    DEFAULT_TIMEOUT_FOR_SERVER_LAUNCH,
+    DEFAULT_URL_FOR_TEST,
+    popen_launch_server,
+)
+
+
+def parse_models(model_string):
+    return [model.strip() for model in model_string.split(",") if model.strip()]
+
+
+class TestEvalAccuracyLarge(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.model_groups = [
+            (parse_models(DEFAULT_MODEL_NAME_FOR_NIGHTLY_EVAL_TP1), False, False),
+            (parse_models(DEFAULT_MODEL_NAME_FOR_NIGHTLY_EVAL_TP2), False, True),
+            (parse_models(DEFAULT_MODEL_NAME_FOR_NIGHTLY_EVAL_FP8_TP1), True, False),
+            (parse_models(DEFAULT_MODEL_NAME_FOR_NIGHTLY_EVAL_FP8_TP2), True, True),
+        ]
+        cls.base_url = DEFAULT_URL_FOR_TEST
+        cls.process = None
+        cls.eval_process = None
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.process:
+            kill_child_process(cls.process.pid)
+        if cls.eval_process:
+            kill_child_process(cls.eval_process.pid)
+
+    def launch_server(self, model, is_fp8, is_tp2):
```

Comment:
try to reuse this from test_nightly_gsm8k_eval.py

Reply:
Ok. BTW I found that there is a bit of an issue with the current human eval for Llama 3.1 8B and 70B, and I need to take some time to identify the cause.

Reply:
It can be written as a static method, I'll change it in the follow-up.